src/components/Shared/BetForm.tsx

```diff
@@ -48,17 +57,29 @@ const BetForm = (props: GamePanelProps) => {
         <>
             <p>
                 Tipo de apuesta:
-                <select className="full-width">
-                    <option selected disabled={true}>Cl&aacute;sica (3 puntos)</option>
+                <select className="full-width" value={betType} onChange={changeBetType}>
+                    <option value="CLASSIC">Cl&aacute;sica (3 puntos)</option>
```

Comment:
Let's add the "please select an option" for the type.

```suggestion
                    {!betType && <option value="" disabled={true}>Seleccione una opci&oacute;n</option>}
                    <option value="CLASSIC">Cl&aacute;sica (3 puntos)</option>
```